.eslintrc.json

```diff
@@ -733,6 +733,8 @@
 					"angular2-grid",
 					"html-query-plan",
 					"turndown",
+					"gridstack",
```

Comment:
While you're here would you mind fixing the tab indentation? Seems like we have a mix of tabs and spaces going on here

Reply:
Sure! I ran Format Document which fixed all the indentations.